src/main/java/com/dataloom/client/serialization/SerializationConstants.java

```diff
@@ -36,8 +36,10 @@ private SerializationConstants() {}
     public static final String ENTITY_DEFINITIONS_FIELD      = "entityDefinitions";
     public static final String ASSOCIATION_DEFINITIONS_FIELD = "associationDefinitions";
 
+    public static final String ACL_KEY                       = "aclKey";
```

Comment:
Right. I was wondering whether or not to use `ACL_OBJECT_PATH`, and I decided it would be better to introduce a duplicate but less confusing `ACL_KEY` constant, since that's what we've been referring to it exclusively.

```
@JsonProperty( SerializationConstants.ACL_OBJECT_PATH ) List<UUID> aclKey
```